fonts.css

```diff
@@ -0,0 +1,27 @@
+
+@font-face {
+    font-family: 'PT Serif';
+    src:
+      url('./fonts/PTSerif-Regular.woff2') format('woff2'),
```

Comment:
Codacy found an issue: [Expected indentation of 4 spaces (indentation)](https://app.codacy.com/gh/WebHeroSchool/VeronikasBaliProject/pullRequest?prid=5807126)